src/pe-parse-1-fixes.patch

```diff
@@ -0,0 +1,43 @@
+This file is part of MXE. See LICENSE.md for licensing information.
+
+Contains ad hoc patches for cross building.
+
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jonas Kvinge <jonas@jkvinge.net>
+Date: Fri, 1 May 2020 12:45:59 +0200
+Subject: [PATCH 1/1] Fix compile
+
+
+diff --git a/cmake/compilation_flags.cmake b/cmake/compilation_flags.cmake
+index 1111111..2222222 100644
+--- a/cmake/compilation_flags.cmake
++++ b/cmake/compilation_flags.cmake
+@@ -22,9 +22,9 @@ else ()
+ 
+     -pedantic -Wall -Wextra -Wcast-align -Wcast-qual -Wctor-dtor-privacy -Wdisabled-optimization
+     -Wformat=2 -Winit-self -Wlong-long -Wmissing-declarations -Wmissing-include-dirs -Wcomment
+-    -Wold-style-cast -Woverloaded-virtual -Wredundant-decls -Wshadow -Wsign-conversion
++    -Wold-style-cast -Woverloaded-virtual -Wno-redundant-decls -Wshadow -Wsign-conversion
+     -Wsign-promo -Wstrict-overflow=5 -Wswitch-default -Wundef -Werror -Wunused -Wuninitialized
+-    -Wno-missing-declarations -Wno-strict-overflow
++    -Wno-missing-declarations -Wno-strict-overflow -fpermissive
```

Comment:
`-fpermissive` is not something we'd merge.

Reply:
I've left it there by a mistake, it compiles fines without. I'll remove it.